README.md

```diff
@@ -1,12 +1,13 @@
 # WSL
 # Text file information reader 
 This program will display the following information about a given text file:
--File size
--Number of characters
--Creation date 
--Last modification date
-To achieve this, we will need to do the following steps:
--Implement reading parameters from the command line - we need to read filepath
--Load the file corresponding to the filepath
--Read file metadata - we need the file creation date, file modification date, the file size
--Read file contents - number of characters, maybe we could read the amount of words in the future as well
\ No newline at end of file
+<p><em>-File size</em></p>
+<p><em>-Number of characters</em></p>
+<p><em>-Creation date</em></p> 
+<p><em>-Last modification date</em></p>
```

Comment:
That the way too, but Markdown has some special syntax which is IMO a bit prettier than plain HTML:
```suggestion
 * File size
 * Number of characters
 * Creation date 
 * Last modification date
```

You can even change it to check box like so:
```suggestion
 * [ ] File size
 * [ ] Number of characters
 * [ ] Creation date 
 * [ ] Last modification date
```